test/conformance/tests/sriov_operator.go

```diff
@@ -59,6 +60,9 @@ func init() {
 	if err == nil && newTime != 0 {
 		waitingTime = time.Duration(newTime) * time.Minute
 	}
+	if isSnoCluster() {
+		snoTimeout = 1
```

Comment:
Can you just have a global timeout that gets overridden in case of sno (as opposed to using 0 / 1 as multiplier)?
Also, I'd have that overrideable with an env variable

Reply:
The timeout will be different in for various tests, so we can't really have a predefined timeout value for SNO.

Reply:
Can you rename the variable as "timeoutMultiplier" or something along that line?

Reply:
Done